### PR TITLE
Update diagnostic shading overrides

### PR DESCRIPTION
**Pull request by Mango-3**
Exposes `Objects`, `Assemblies` and `Screen-Space Velocity` diagnostic surface shader.

### CODE REVIEW

File: src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp [hunk 1]
```diff
@@ -213,26 +213,29 @@ namespace
...
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_15,        L"Original Shading Normals" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_16,        L"Primitives" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_17,        L"Ray Spread" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_18,        L"Screen-Space Velocity" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_19,        L"Screen-Space Wireframe" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_20,        L"Shading Normals" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_21,        L"Sides" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_22,        L"Tangents" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_23,        L"UV Coordinates" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_24,        L"World Space Position" },
```
Comment (dictoon) [inline src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp]: For consistency's sake, "World-Space" (below as well).

File: src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp [hunk 2]
```diff
@@ -154,26 +154,29 @@ const char* get_shader_override_type(const int shader_override_type)
...
-      case 14: return "ray_spread";
-      case 15: return "screen_space_wireframe";
-      case 16: return "shading_normal";
-      case 17: return "sides";
-      case 18: return "tangent";
-      case 19: return "uv";
-      case 20: return "world_space_position";
-      case 21: return "world_space_velocity";
-      case 22: return "world_space_wireframe";
+      case 3:  return "assemblies";
```
Comment (dictoon) [inline src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp]: We're breaking compatibility. I'm fine with that given that these are just diagnostic modes, but I just want to make sure we're aware of that :)

File: src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp (hunk 1 above)
Comment (Mango-3) [reply src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp]: indeed

File: src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp (hunk 2 above)
Comment (Mango-3) [reply src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp]: Yep, I'm aware of this.

**Comment by dictoon** [comment]
Thanks!

**Pull request merged**